test/runtests.jl

```diff
@@ -1,3 +1,14 @@
+
+# FIXME: Once StyledStrings v1 is registered, remove the following and the `Pkg` dependency
+if isdefined(Base, :get_extension)
+    import Pkg
+    if VERSION < v"1.11"
+        Pkg.add(url="https://github.com/JuliaLang/StyledStrings.jl", rev="julia1-compat")
```

Comment:
I don't know how long `julia1-compat` remains in effect, but it is more practical than `main`.

Reply:
I don't anticipate deleting the branch at any point, it will just become abandoned at some stage.